src/Bicep.Core/Syntax/TargetScopeSyntax.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using Bicep.Core.Navigation;
+using Bicep.Core.Parser;
+using Bicep.Core.TypeSystem;
+
+namespace Bicep.Core.Syntax
+{
+    public class TargetScopeSyntax : SyntaxBase, IDeclarationSyntax
+    {
+        public TargetScopeSyntax(Token keyword, Token assignment, SyntaxBase value)
```

Comment:
>Token [](start = 48, length = 5)

Following the comment in the Parser, this should be SyntaxBase with a corresponding multi-type assertion below.